Lib/test/bad_delattr.py

```diff
@@ -0,0 +1,2 @@
+foo = 1
+__delattr__ = "Oops"
```

Comment:
Can you create a new Lib/test/test_import/ sub-directory for these new files? You must then add the directory to TESTSUBDIRS in Makefile.pre.in.

Reply:
@vstinner, the current layout was like PEP 562, see https://github.com/python/cpython/pull/4731.  Should we move these files as well?